### DISCUSSION TITLE
Sentry + Quasar

### DISCUSSION THREAD

**Discussion by ghosteye1** [quasarframework/quasar#9331] [Ideas / Proposals] [answered]
Hello, Anyone ever had the experience of working with Quasar with Sentry to catch errors.
Which practice would you recommend?

**Comment by ontwikkelfabriek** [comment, marked as answer]
Yes I have it running on some of my Quasar projects. 

I would start here:
https://docs.sentry.io/platforms/javascript/guides/vue/

You have to use a boot file for the Sentry `init` code.
https://quasar.dev/quasar-cli/boot-files

My Sentry boot file in `src/boot/sentry.js`:
```javascript
import Vue from 'vue'
import * as Sentry from '@sentry/browser'
import * as Integrations from '@sentry/integrations'



if (process.env.PROD) {
    let options = {
        release: '0.0.1',
        dsn: 'https://XXXXXXXXXXXXXXXXXXX@sentry.io/XXXXXX',
        integrations: [new Integrations.Vue({ Vue, attachProps: true })],
        ignoreErrors: ['ResizeObserver loop limit exceeded']
    }
    Sentry.init(options)
}

```

**Reply by ghosteye1** [reply]
Thank you. How did you add into quasar.conf boot section?

**Reply by ontwikkelfabriek** [reply]
> Thank you. How did you add into quasar.conf boot section?> 

Like this for the `src/boot/sentry.js` boot file:
```
 boot: [ ... ,  'sentry', ... ]
```

**Reply by ghosteye1** [reply]
Thank you very much
